gen-certs.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/sh
+
+set -e
+
+# These commands are correct for generating certificates using openssl 1.1.1l
+# Check your version by running `openssl version` and if it's LibreSSL, install
+# the proper version of openssl using a tool like homebrew
+
+VERSION=$(openssl version)
+
+if [[ "$VERSION" == *"Libre"* ]]; then
+	echo "This script is not compatable with LibreSSL."
+	exit
+fi
+
+while getopts o: flag
+do
+    case "${flag}" in
+        o) output_location=${OPTARG};;
+    esac
+done
+
+if [ -z "$output_location" ]; then
+	output_location=.
+fi
+echo $output_location
+
+mkdir -p $output_location
+
+temp=$output_location/temp
+mkdir -p $temp
```

Comment:
Because you're using `mkdir -p` here, you can skip the `mkdir` above, it will create all necessary directories in the path.